recipes/piqs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "piqs" %}
+{% set version = "1.2" %}
+{% set sha256 = "2be0f330ebf5cdad5c353c763d683054f09c6a9d08f8b34a71a056c29755cd75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathanshammah/piqs/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - numpy
+    - cython >=0.21
+  
```

Comment:
add 
```
 - msinttypes  # [win]
```

To the build section you probably also need 

```
  features:
    - vc9  # [win and py27]
    - vc14  # [win and (py35 or py36)]
```

Easiest way to skip windows builds is with

```
build:
   skip: True  #[win]
```